src/models/Transaction.js

```diff
@@ -174,11 +177,11 @@ export type Transaction = {
 export type TransactionPayload = {
   gasLimit?: number,
   to: string,
-  receiverEnsName?: string,
+  receiverEnsName?: ?string,
```

Comment:
How about either present as valid or not present at all?

Reply:
I've updated the type to reflect true potential values with the current code.